infrastructure-playbooks/rolling_update.yml

```diff
@@ -224,7 +224,7 @@
     - name: fetch ceph mgr key(s)
       fetch:
         src: "{{ ceph_conf_key_directory }}/{{ cluster }}.mgr.{{ hostvars[item]['ansible_hostname'] }}.keyring"
-        dest: "{{ fetch_directory }}/{{ fsid }}/"
+        dest: "{{ fetch_directory }}/{{ fsid }}/{{ ceph_conf_key_directory }}"
```

Comment:
"msg": "dest is an existing directory, use a trailing slash if you want to fetch src into that directory"

Reply:
true. good catch, thanks @yprokule